plugins/httpapi/nd.py

```diff
@@ -337,6 +367,13 @@ def _verify_response(self, response, method, path, data):
         #     self.info['msg'] = msg
         self.info["body"] = response_data
 
+        # Set info keys required for cisco.dcnm modules
+        self.info["RETURN_CODE"] = response_code
```

Comment:
I'm not sure about the behaviour but will this info be displayed for ND modules as well? If that's the case shall we segregate what's displayed for ND and NDFC by leveraging platform?

Reply:
Based on my testing and review of our code, I don't believe we use the `send_request()` functions in a way that surfaces the entire `info` dictionary. We use the specific keys from the returned `info` instead. The new ones required by `ansible-dcnm` will be ignored.

Reply:
The goal was to make the `ansible-nd` HTTPAPI plugin compatible without changes to `ansible-dcnm`. I am not sure if there is way to know which platform/module is being used without making more changes.